def raise_exception():
    raise TypeError("my error")
